README.md

```diff
@@ -38,6 +38,7 @@ Flags:
                                    Std | Standard - Captures all standard packages if they do not match another section
                                     (default [Standard,Default])
   -x, --SectionSeparator strings   SectionSeparators are inserted between Sections (default [NewLine])
+      --SkipGeneratedFiles         Don't process generated files
```

Comment:
I recommend that add string array as params here to make users easier do custom skip.